03_probability_intro/03_probability_intro.jl

```diff
@@ -13,126 +13,338 @@ macro bind(def, element)
+
+Then we represent the event A , that contains the 3 sample points $\{(R,W), (G,W)$ and $(B,W)\}$, with a red circle.
+
+The event consisting of all points not contained in the event A will be called the complementary event (or negation) of A and is denoted by A'.
+
+$A´= {(R,G),(R,B),(G,B)}$
+
+Notice that if we created a new event that contains all the sample points of A and A´, we obtain the sample space.
+
+"
+
+
+# ╔═╡ 98d90100-919a-11eb-04cd-4729037de830
+imresize(load("images/venn-3.jpg"), (336, 400))
+
+# ╔═╡ d8e086b8-9241-11eb-33f2-03786ca67fa1
+md"
+Now let's take into play the event B.
+The points $(G,W)$ and $(B,W)$ are present in the event A and B. So we must represent them in this way:
+"
+
+# ╔═╡ 985fcce0-919a-11eb-0fa6-9d8ca380ce4e
+imresize(load("images/venn-2.jpg"), (336, 400))
+
+# ╔═╡ 75091186-9242-11eb-1fdf-fddc652f1453
+md"
+The blue area contains the sample points that are in both events, in this case $(G,W)$ and $(B,W)$. Is defined as the intersection of A and B and is denoted by $A \cap B$.
+
+On the other hand, the red area contains the points that are only present in the event A, in this case $(R,W)$ .
+Analogously the green area contains the points that are only present in the event A, in this case $(G,B)$ .
+"
+
+# ╔═╡ cf075ae6-922f-11eb-1ff2-b70737c34f82
+md"""
+## Probability
+
+Now that we are introduced to the event, sample point and sample space concepts we can start talking about probability.
+
+Probability is a measure of our belief that a particular event will occur, and we express this with a number ranging from $0$ to $1$.
 The number $0$ means we have the strongest possible belief that the event will not happen: We are sure it will not happen.
 The number $1$ means we have the strongest possible belief that the event will happen: We are sure it will happen. 
 Probability, being a measure of our own belief or certainty in the occurrence of an event, does not determine whether the event occurs or not. 
 For this reason, events may still occur when we assign them probability $0$, and they might not occur if we assign them probability $1$.
-How exactly this number is linked to each event is something that is far from trivial and indeed is a discussion that has been going for years. Later on we'll dive deeper into that rabbit hole, but for the moment let's not worry about how exactly this is defined or calculated: assume we have a well established method for the time being. The important thing now will be the different rules that emerge from the nature of these events.
+
+By definition, the probability of the entire sample space $S$ is unity, or $P\{S\} = 1$. 
+It follows that for any event $A$: $0 <P(A) <1$.
+
+In our experiment we can consider that all the sample points have the same probability of going out so we assign ⅙ to each point.
+Another way we can assign probabilities to each sample point is with the popular formula:
```

Comment:
replace with:
"Since we consider that all the sample points have the same realization probability, another way we can assign probabilities to each one is with the popular formula:"